editors/code/src/commands/inlay_hints.ts

```diff
@@ -0,0 +1,100 @@
+import * as vscode from 'vscode';
+import {
+    Range,
+    TextDocumentChangeEvent,
+    TextEditor
+} from 'vscode';
```

Comment:
```ts
import { Range, TextDocumentChangeEvent, TextEditor } from 'vscode';
```

Reply:
Do you get this error from prettier locally? How it works? I don't have this error myself...

Reply:
Yes. It's not an error, I just run it followed by git diff.

Reply:
@SomeoneToIgnore can you try making this change?

Reply:
Thanks for the tip, I'm trying that.